File: bubble_sort.py
class bubble_sort:
    def __init__(self,array):
        for x in range(len(array)-1):
            for y in range(len(array)-1):
                if array[y] > array[y + 1]:
                    temp = array[y]
                    array[y] = array[y + 1]
                    array[y + 1] = temp
                    print(array)
array=[7,8,2,3,4]       
bubble_sort(array)                    

File: bubble_sort.java
//
class Coded {
    
    public void sort(int x[]){
        for (int k = 0; k < x.length - 1; k++) {
            for (int i = 0; i < x.length-1; i++) {
                if (x[i] > x[i + 1]) {
                   int temp = x[i];
                    x[i] = x[i + 1];
                    x[i + 1] = temp;
                    
                }
            }
            
        }
   //iterating elements of  array
        for (int i : x) {
            System.out.println("\n"+i); 
        }
         

    
   }

    
}

public class bubble_sort {
    
    public static void main(String[] args) {
        Coded code = new Coded();
        //testing
       int values[] = {19,5,6,4,2,9,0};
         code.sort(values);
        
      
    }}
